docs/src/content/docs/getting-started/usage.mdx

```diff
@@ -2,24 +2,90 @@
 title: Usage
 description: Learn how to use astro-integration-kit in your project.
 ---
+import Disabled from "~/components/Disabled.astro"
+import { LinkCard, Tabs, TabItem } from '@astrojs/starlight/components';
 
 Astro Integration Kit can be used in 2 ways: as **extended hooks** or as **standalone utilities**.
 
+## Which flavor should you use?
+
+**Extended hooks** provide a way better DX and we recommend picking this one by default. However, if you have
```

Comment:
Tempted to not even call this "extended hooks" any more... I feel like it's not even really that anymore, it's just a nice modular opt in dealy.

Instead of "extended hooks" it probably should just be like "using Astro Integration Kit Plugins" or something, like if we're calling them plugins when used all-in, then utilities when standalone. Mays as well call them plugins here. But I'll level with you, that's a lot of work rn and can be updated later when we actually have something better to put

Reply:
Yeah I did more docs than anticipated so I think we can keep this as is for now, then update the utilities docs with more content then work on terminology once more